testing/src/cypress.env.json.sample

```diff
@@ -0,0 +1,22 @@
+{
+  "ENVIRONMENT": "TEST",
+  "TEST": {
+    "BASE_URL": "https://test-aest-sims.apps.silver.devops.gov.bc.ca",
+    "KEY_CLOAK_URL": "https://test.oidc.gov.bc.ca/auth/realms/jxoe2o46/protocol/openid-connect/token",
+    "USERNAME_SINGLE_LOCATION": "QASIMS_Collf",
+    "USER_PASSWORD_SINGLE_LOCATION": "",
+    "USERNAME_API_TEST":"4116796724b34ebd98dd035a9e4570c0@bceid",
+    "USER_PASSWORD_API_TEST":""
+  },
+  "DEV": {
+    "BASE_URL": "https://dev-aest-sims.apps.silver.devops.gov.bc.ca",
+    "KEY_CLOAK_URL": "https://dev.oidc.gov.bc.ca/auth/realms/jxoe2o46/protocol/openid-connect/token"
+  },
+  "END_POINTS": {
```

Comment:
I would say that endpoint is a one-word only.

Reply:
updated